servers.json

```diff
@@ -37,6 +37,16 @@
             "glade.gg"
         ]
     },
+    {
+        "id": "pvpbeach",
+        "name": "PvPBeach Network",
+        "addresses": [
+            "pvpbeach.com"
+        ],
+        "primaryAddress": "pvpbeach.com",
+        "minecraftVersions": ["1.7", "1.8", "1.9", "1.10", "1.11", "1.12"],
```

Comment:
Only the following versions are supported by Lunar:
"1.7", "1.8", "1.12", "1.16", "1.17", "1.18", "1.19"

Please remove 1.9, 1.10, and 1.11.